app/processing/delta/split-to-ledger.js

```diff
@@ -39,7 +39,7 @@ const copyPaymentRequest = (paymentRequest, ledger) => {
 
 const calculateInvoiceLines = (invoiceLines, apportionmentPercent) => {
   invoiceLines.map(x => {
-    x.value = x.value > 0 ? Math.ceil(x.value * apportionmentPercent) : Math.floor(x.value * apportionmentPercent)
+    x.value = Math.trunc(x.value * apportionmentPercent)
```

Comment:
We are working in pence so no need to do apply any rounding.  Can safely just drop decimals.  Applying rounding will result in incorrect line deltas